src/main/java/uk/ac/ebi/eva/pipeline/parameters/JobOptions.java

```diff
@@ -37,11 +37,10 @@
 public class JobOptions {
     private static final Logger logger = LoggerFactory.getLogger(JobOptions.class);
 
-    @Value("${" + JobParametersNames.APP_OPENCGA_PATH + "}") private String opencgaAppHome;
+    @Value("${" + JobParametersNames.APP_OPENCGA_PATH + ":#{null}}") private String opencgaAppHome;
 
     // Pipeline application options.
     @Value("${" + JobParametersNames.CONFIG_RESTARTABILITY_ALLOW + ":false}") private boolean allowStartIfComplete;
```

Comment:
This is not the same parameter. It is the old parameter to set the restartability of steps. I haven't removed it until everyone is comfortable with it.